apps/core/Search.js

```diff
@@ -1,58 +1,82 @@
 // @flow
 
 import * as React from 'react';
-import { StyleSheet, View, Button } from 'react-native';
-import { QueryRenderer, graphql } from '@kiwicom/margarita-relay';
-import { Text } from '@kiwicom/margarita-components';
+import { StyleSheet, View, Button, TextInput } from 'react-native';
 
 import Routes from './config/routes';
-import { type SearchQueryResponse } from './__generated__/SearchQuery.graphql';
 
 type Props = Object;
+type State = Object;
 
-export default class Search extends React.Component<Props> {
-  goToResults = () => {
-    this.props.navigation.navigate(Routes.RESULTS);
+export default class Search extends React.Component<Props, State> {
+  constructor(props: Props) {
+    super(props);
+    this.state = {
+      travelFrom: 'OSL',
+      travelTo: 'PRG',
+      dateFrom: '2018-12-30',
+      dateTo: '2018-12-31',
```

Comment:
Why are these values here?

Reply:
It's probably because we don't have the PlacePicker and DatePicker yet, so at least we can see how the app works?

Reply:
They are currently here as sample default values with plan to remove them in next iteration when adding proper inputs and look.
But I can definitely remove them even now.